registry/kubevirt/0.0.1/virtoperator.clusterrole.yaml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
this needs to be vendored from kubevirt as well.

Basically any manifest that comes from kubevirt, cdi, etc... needs to be vendored from those projects. That's going to be the best way to keep all of this in sync properly.